src/internal/dbutil/tx.go

```diff
@@ -203,3 +203,23 @@ func isTransactionError(err error) bool {
 	pgxErr := &pgconn.PgError{}
 	return errors.As(err, &pgxErr) && strings.HasPrefix(pgxErr.Code, "40")
 }
+
+// DoTx is a shorter alias for WithTx
+func DoTx(ctx context.Context, db *pachsql.DB, fn func(ctx context.Context, tx *pachsql.Tx) error, opts ...WithTxOption) error {
```

Comment:
Hmm I don't see this new function being used anywhere. Is the plan to use this or `DoTx1` in the future instead of `WithTx`?

Reply:
This was on another branch.  I don't know how it got in here.  Good catch.  I'll remove.